hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommits.java

```diff
@@ -151,13 +151,13 @@ public void testThriftExceptionSuccessOnCommit() throws TException, InterruptedE
     spyOps.commit(metadataV2, metadataV1);
 
     ops.refresh();
-    Assert.assertNotEquals("Current metadata should have changed", metadataV2, ops.current());
-    Assert.assertTrue(
-        "Current metadata file should still exist", metadataFileExists(ops.current()));
-    Assert.assertEquals(
-        "Commit should have been successful and new metadata file should be made",
-        3,
-        metadataFileCount(ops.current()));
+    Assertions.assertThat(ops.current()).as("Current metadata should have changed").isNotEqualTo(metadataV2);
+    Assertions.assertThat(
+        metadataFileExists(ops.current())).as( "Current metadata file should still exist").isTrue();
+    Assertions.assertThat(
+
```

Comment:
why the newline here?

Reply:
Removed. That came due to the Junit5 migration script